metecho/tests/consumers.py

```diff
@@ -247,6 +248,6 @@ async def test_push_notification_consumer__missing_instance():
         "id": "bet this is an invalid ID",
         "payload": {},
     }
-    consumer = PushNotificationConsumer({})
+    consumer = PushNotificationConsumer()
     new_content = await consumer.hydrate_message(content)
     assert new_content == {"payload": {}}
```

Comment:
I assume these changes are related to a new version of django-channels?

Reply:
Yep